keyboards/business_card/alpha/keymaps/default/keymap.c

```diff
@@ -0,0 +1,136 @@
+/* Copyright 2019 kakunpc
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+#include "lufa.h"
+#include "ssd1306.h"
+#include "rgblight_list.h"
+
+// Defines the keycodes used by our macros in process_record_user
+enum custom_keycodes {
+    HOME = SAFE_RANGE,
+    NAME,
+    ADDRESS,
+    SKILL,
+    SOCIAL,
+    ABOUT
+};
+
+const uint16_t PROGMEM
+keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+[0] =
+LAYOUT(/* Base */
+        SKILL, SOCIAL, ABOUT, \
+        HOME, NAME, ADDRESS
+),
+};
+
+int type = HOME;
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    switch (keycode) {
+        case NAME:
+            if (record->event.pressed) {
+                type = NAME;
+                rgblight_sethsv_blue();
```

Comment:
Especially if you're doing this often, you DO NOT want to use the EEPROM version. 

```suggestion
                rgblight_sethsv_noeeprom_blue();
```